src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/PiattaformaNotifichePGPAPage.java

```diff
@@ -53,8 +53,14 @@ public void waitLoadPitattaformaNotificaPage(String ragioneSociale) {
     }
 
     public void clickSuDelegeButton() {
-        this.getWebDriverWait(30).until(ExpectedConditions.visibilityOf(this.delegheSideMenu));
-        this.js().executeScript("arguments[0].click()",this.delegheSideMenu);
+        try {
+            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOf(this.delegheSideMenu));
```

Comment:
inserire messaggio

Reply:
già gestito da try and catch non serve metterlo nel withmessage